server.js

```diff
@@ -109,6 +137,16 @@ async function searchByCountry(server) {
   }
 }
 
+//adds the search to history table
+async function addSearchToHistory(server, country, indicator, year) {
+  const user_id = await getCurrentUser(server);
+  if (user_id) {
+    db.query(
+      `INSERT INTO history (user_id, country_name, indicator, year) VALUES (?, ?, ?, ?)`,
+      [user_id, country, indicator, year]
```

Comment:
Everything looks great. I think it would be worth storing the date the search was created at too. That way we could display the search history in order back on our front end